src/dmd/dtemplate.d

```diff
@@ -559,6 +559,7 @@ extern (C++) final class TemplateDeclaration : ScopeDsymbol
     bool ismixin;           // this is a mixin template declaration
     bool isstatic;          // this is static template declaration
     bool isAliasSeq;        /// matches `template AliasSeq(T...) { alias AliasSeq = T; }
+    bool isAlias;           /// matches `template Alias(T) { alias Alias = T; }
```

Comment:
Also this comment odes not match what is in the PR description

Reply:
`template Alias(T) { alias Alias = T; }` matches it.

Reply:
I meant as in 
```
template InoutOf(T) { alias InoutOf       =        inout(T) ; }
template ConstOf(T) { alias ConstOf       =        const(T) ; }
template SharedOf(T) { alias SharedOf      =       shared(T) ; }
template SharedInoutOf(T) { alias SharedInoutOf = shared(inout(T)); }
template SharedConstOf(T) { alias SharedConstOf = shared(const(T)); }
template ImmutableOf(T) { alias ImmutableOf   =    immutable(T) ; }
```

Even if it technically does, that is not immediately obvious.

Reply:
Yeah I think `isAliasLike` or `isTrivialAliasing` would convey the meaning much better.
I was originally under the impression that the name would be matched, originally.